pkg/loadtest/local_load_test.go

```diff
@@ -98,7 +98,7 @@ func TestLocalLoad(t *testing.T) {
 	})
 	auditChecker := version.NewDefaultAuditChecker(conf.GetSecurityAuditCheckURL())
 	emailParams, _ := conf.GetEmailParams()
-	emailer := email.NewEmailer(emailParams)
+	emailer, _ := email.NewEmailer(emailParams)
```

Comment:
verify the return error - this is a valid flow that can fail

Reply:
fixed